grafana-plugin/src/network/network.ts

```diff
@@ -1,5 +1,3 @@
-import { SpanStatusCode } from '@opentelemetry/api';
```

Comment:
we should replicate those changes in grafana-plugin/src/network/oncall-api/http-client.ts

Reply:
done